tests/e2e/tests/pilot/routing_test.go

```diff
@@ -334,3 +325,37 @@ func TestRouteRedirectInjection(t *testing.T) {
 		}()
 	}
 }
+
+func TestRouteMirroring(t *testing.T) {
+	t.Skipf("Skipping %s", t.Name())
```

Comment:
You're always skipping this test?

Reply:
At least add a TODO?

Reply:
Yes.. It seems ths test always passes and I am not sure if I did it right.